package.json

```diff
@@ -104,7 +104,7 @@
     "@mapbox/query-selector-contains-node": "^1.0.0",
     "@mapbox/scroll-restorer": "^1.0.0",
     "address": "^1.1.2",
-    "assets-webpack-plugin": "3.9.12",
```

Comment:
I was tinkering with dependencies, and realized we can add a caret back, since the regression that made us set the version (https://github.com/mapbox/batfish/pull/298) has been fixed.